contracts/utils/FuseFixer.sol

```diff
@@ -0,0 +1,189 @@
+        address(0xB7FE5f277058b3f9eABf6e0655991f10924BFA54), // Pool 18: USTw
+        address(0x9de558FCE4F289b305E38ABe2169b75C626c114e), // Pool 27: FRAX
+        address(0xda396c927e3e6BEf77A98f372CE431b49EdEc43D), // Pool 27: FEI
+        address(0xF148cDEc066b94410d403aC5fe1bb17EC75c5851), // Pool 27: ETH
+        address(0x0C402F06C11c6e6A6616C98868A855448d4CfE65), // Pool 27: USTw
+        address(0x26267e41CeCa7C8E0f143554Af707336f27Fa051), // Pool 127: ETH
+        address(0xEbE0d1cb6A0b8569929e062d67bfbC07608f0A47), // Pool 127: USDC
+        address(0x4B68ef5AB32261082DF1A6C9C6a89FFD5eF168B1), // Pool 127: DAI
+        address(0xe097783483D1b7527152eF8B150B99B9B2700c8d), // Pool 127: USDT
+        address(0x0F0d710911FB37038b3AD88FC43DDAd4Edbe16A5), // Pool 127: USTw
+        address(0x8922C1147E141C055fdDfc0ED5a119f3378c8ef8), // Pool 127: FRAX
+        address(0x7DBC3aF9251756561Ce755fcC11c754184Af71F7), // Pool 144: ETH
+        address(0x3a2804ec0Ff521374aF654D8D0daA1d1aE1ee900), // Pool 144: FEI
+        address(0xA54c548d11792b3d26aD74F5f899e12CDfD64Fd6), // Pool 144: FRAX
+        address(0xA6C25548dF506d84Afd237225B5B34F2Feb1aa07), // Pool 144: DAI
+        address(0xfbD8Aaf46Ab3C2732FA930e5B343cd67cEA5054C), // Pool 146: ETH
+        address(0x49dA42a1EcA4AC6cA0C6943d9E5dc64e4641e0E3), // Pool 146: wstETH
+        address(0xe14c2e156A3f310d41240Ce8760eB3cb8a0dDBE3), // Pool 156: USTw
+        address(0x001E407f497e024B9fb1CB93ef841F43D645CA4F), // Pool 156: FEI
+        address(0x5CaDc2a04921213DE60B237688776e0F1A7155E6), // Pool 156: FRAX
+        address(0x9CD060A4855290bf0c5aeD266aBe119FF3b01966), // Pool 156: DAI
+        address(0x74897C0061ADeec84D292e8900c7BDD00b3388e4), // Pool 156: LUSD
+        address(0x88d3557eB6280CC084cA36e425d6BC52d0A04429), // Pool 156: USDC
+        address(0xe92a3db67e4b6AC86114149F522644b34264f858) // Pool 156: ETH
+    ];
+
+    mapping(address => address[]) internal underlyingsToCTokens;
+
+    constructor(address core) CoreRef(core) {
+        _buildCTokenMapping();
+
+        // check mappings lengths - hand calculated
+        assert(underlyingsToCTokens[UNDERLYINGS[0]].length == 7);
+        assert(underlyingsToCTokens[UNDERLYINGS[1]].length == 5);
+        assert(underlyingsToCTokens[UNDERLYINGS[2]].length == 5);
+        assert(underlyingsToCTokens[UNDERLYINGS[3]].length == 1);
+        assert(underlyingsToCTokens[UNDERLYINGS[4]].length == 5);
+        assert(underlyingsToCTokens[UNDERLYINGS[5]].length == 3);
+        assert(underlyingsToCTokens[UNDERLYINGS[6]].length == 2);
+        assert(underlyingsToCTokens[UNDERLYINGS[7]].length == 2);
+        assert(underlyingsToCTokens[UNDERLYINGS[8]].length == 4);
+        assert(underlyingsToCTokens[UNDERLYINGS[9]].length == 1);
+
+        // send out token approvals
+        _approveCTokens();
+    }
+
+    /// @dev Repay all debt
+    function repayAll() public onlyTribeRole(TribeRoles.PCV_GUARDIAN_ADMIN) {
```

Comment:
Think this is the plan anyway, but much more in favour of partially repaying and checking progress/status as we go